debian/copyright

```diff
@@ -0,0 +1,90 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: dragon
```

Comment:
This is the wrong package. Similarly to my question at the SDB repo - is this necessary?